docs/.vitepress/Footer.vue

```diff
@@ -48,14 +51,14 @@ const baseUrl = import.meta.env.BASE_URL;
   height: 104px;
   background: url('stars-background.svg') no-repeat center/cover;
 }
-
+/*
```

Comment:
Hm. Is this commented out code needed here? I think it's part of the previous [WIP: style front-page, header and footer PR](https://github.com/warp-ds/warp-portal-poc/pull/10), right?